src/Feed/KobaFeedType.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace App\Feed;
```

Comment:
Have we given any thought on how to namespace/install custom feeds? KOBA is specific to Aarhus....

Reply:
I think this could be solved by just adding a bundle where the feed types implement FeedTypeInterface.
https://github.com/os2display/display-api-service/blob/develop/config/services.yaml#L19
The issue is how to be able to access this interface from the bundle.